include/seqan3/range/views/to.hpp

```diff
@@ -0,0 +1,26 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
+ * \brief Provides seqan3::views::to.
+ */
+
+#pragma once
+
+#include <range/v3/view/zip.hpp>
+
+#include <seqan3/core/platform.hpp>
+#include <seqan3/std/ranges>
+
+namespace seqan3::views
+{
+
+//TODO reimplement me
+using ::ranges::to;
```

Comment:
ranges::to is not in the view namespace of the ranges library. There must be a rational for this. Why are you breaking it?

Reply:
Well, this is a discussion point I still had. The ranges library places it in `std::ranges::to`, but we don't have a ranges namespace and naming it `seqan3::to` seems too generic for me.

Also we have the notion of "views", "source-only-views", "sink-only-views" (`to` falls into the last category); the range library calls these "view adaptors", "view factories" and has no name for the last. 

Since `to` can only be applied to views, I would put it in `seqan3::views::` but we can discuss that tomorrow.

Reply:
Mhmm, I see. I think about it but unfortunately, I won't be there tomorrow. I had to take spontaneous vacation. (also wrote it into the gitter channel, sorry for the circumstances). I think it is a little unintuitive what it does then, also since we have views::convert which converts the range element wise. Do we have a sink-only view? Apparently, by our current definition this would not be a sink-only view since it returns a range 😏. Anyway, I find something like `to_container` or `materialize` more intuitive for the user as it more clearly states the intention.